packages/cli/src/test/compile.test.ts

```diff
@@ -0,0 +1,203 @@
+import {command} from "../lingui-compile"
+import {makeConfig} from "@lingui/conf"
+import path from "path"
+import {getConsoleMockCalls, mockConsole} from "@lingui/jest-mocks"
+import mockFs from "mock-fs"
+import fs from 'fs'
+
+function readFsToJson(directory: string, filter?: (filename: string) => boolean) {
+  const out = {}
+
+  fs
+    .readdirSync(directory)
+    .map((filename) => {
+      const filepath = path.join(directory, filename)
+
+      if (fs.lstatSync(filepath).isDirectory()) {
+        out[filename] = readFsToJson(filepath)
+        return out
+      }
+
+      if (!filter || filter(filename)) {
+        out[filename] = fs.readFileSync(filepath).toString()
+      }
+    })
+
+  return out
+}
+
+describe('CLI Command: Compile', () => {
+  xit('Should return error if no messages', () => {
+    const config = makeConfig({
+      locales: ['en', 'pl'],
+      rootDir: path.join(__dirname, 'fixtures/compile'),
+      catalogs: [{
+        path: '<rootDir>/{locale}',
+        include: ['<rootDir>']
+      }]
+    })
+
+    const result = command(config, {})
+    expect(result).toBeFalsy()
+  })
+
+  describe('Merge Catalogs', () => {
+    // todo
+  })
```

Comment:
@Martin005 Would you be minded to write tests for the latest changes  to the merging functionality introduced by you?

Reply:
We can create issues labeled with the `good first issue` for such cases to encourage the community to contribute